scripted-tests/run/string-from-to-ctring/project/scala-native.sbt

```diff
@@ -1,7 +1,8 @@
 {
   val pluginVersion = System.getProperty("plugin.version")
-  if(pluginVersion == null)
-    throw new RuntimeException("""|The system property 'plugin.version' is not defined.
+  if (pluginVersion == null)
+    throw new RuntimeException(
+      """|The system property 'plugin.version' is not defined.
```

Comment:
This is a bit of a shame, `assumeStandardLibraryStripMargin = true` would fix it.